tembo-operator/src/apis/coredb_types.rs

```diff
@@ -530,8 +589,7 @@ pub struct CoreDBSpec {
     /// backups and WAL archive uploads to an S3 compatible object store.
     ///
     /// **Default**: disabled
-    #[serde(default = "defaults::default_backup")]
-    pub backup: Backup,
+    pub backup: Option<Backup>,
```

Comment:
maybe we roll this back to simplify this PR to not worry about incompatible CRDs